setup.py

```diff
@@ -64,6 +67,12 @@ def get_adccore_data():
 
     import AdcCore
 
+    if is_conda_build():
+        os.environ["LDFLAGS_LD"] = \
+            os.environ["LDFLAGS_LD"].replace("-dead_strip_dylibs", "")
+        os.environ["LDFLAGS"] = \
+            os.environ["LDFLAGS"].replace("-Wl,-dead_strip_dylibs", "")
```

Comment:
Ok this obviously modifies the environment for the full built. Is this exactly what is needed?

Reply:
Yes, because somehow, the destructive flags otherwise get picked up again somehow.
I find this also super annoying and ugly.

Reply:
Me too, but ok. Let's leave it for the time being. Once we manage to redo our build system with the OSS libtensor we can revisit this. Can you add a TODO?